src/Integrations/LaravelServiceProvider.php

```diff
@@ -25,7 +25,11 @@ public function boot()
         ]);
 
         Queue::after(function (JobProcessed $event) {
-            $event->job->delete();
+            if ($event->job->getConnectionName() === 'sqs-plain') {
+                if (!$event->job->isReleased()) {
```

Comment:
```suggestion
                if (!$event->job->isDeletedOrReleased()) {
```

This might be a better method to use, this will then prevent jobs from being deleted twice which can cause some performance issues

Reply:
Good point, thank you